CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 
 ### Changed
 
+- Added Ruby 3.2 to the CI matrix. ([PR#1690](https://github.com/cucumber/cucumber-ruby/pull/1690))
```

Comment:
Minor: extremely minor. Our convention is to use the present tense, as in "Add" is preferred over "Added".

Reply:
Thank you for your quick review. Indeed it is. I updated this PR.